docs/setup/turn/coturn.md

```diff
@@ -0,0 +1,188 @@
+# coturn TURN server
+
+The following sections describe how to install [coturn](<https://github.com/coturn/coturn>) (which implements the TURN REST API).
+
+## `coturn` setup
+
+### Initial installation
+
+The TURN daemon `coturn` is available from a variety of sources such as native package managers, or installation from source.
+
+#### Debian installation
+
+Just install the debian package:
+
+```sh
+apt install coturn
```

Comment:
```suggestion
sudo apt install coturn
```
Usually you are not switched or logged into the root user directly. This seems to be more in line with other documentation for installing software in general.